### ISSUE TITLE
Position: right, caret should be flipped

### ISSUE THREAD

**Issue by boldtrn** [nickpeihl/leaflet-sidebar-v2#28]
If you init a container on the right side, the caret that closes the panel, should be flipped. Right now it looks like it would move further out.

Probably we should consider the position at this point:
```
 if (this.options.closeButton)
                    content += '<span class="leaflet-sidebar-close"><i class="fa fa-caret-left"></i></span>';
```

**Comment by boldtrn** [comment]
Thanks for the reply. This only works if you use html though. If you use the JS API this does not work.

**Comment by noerw** [comment]
fixed in just released v3.0.5

**Comment by boldtrn** [comment]
Thanks :)
